token-client/src/test/resources/x509Base64.txt

```diff
@@ -0,0 +1,35 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
is it like the one you get via vcap services? I'm missing \n linebreaks

Reply:
nope, needs to be changed, remove the header and footer as that's how it looks in the x-forwarded-client-cert header

Reply:
ah, okay, then maybe it should be renamed to cf-forwarded-client-cert.txt or sth like that